README.md

```diff
@@ -51,6 +51,7 @@ To contribute, fork this repository, add your amazing entry and send us a PR. Se
 
 ## Examples
 
+* [30-seconds-of-interviews](https://github.com/fejes713/30-seconds-of-interviews) - A curated collection of common interview questions.
```

Comment:
@dmitrykurmanov Can you change the title to "30 Seconds of Interviews". It looks nicer that way.

Reply:
Done